src/components/elements/table/GenericTable.stories.tsx

```diff
@@ -3,6 +3,7 @@ import { Meta, StoryFn } from '@storybook/react';
 
 import GenericTable, { Props as GenericTableProps } from './GenericTable';
 import { SsnDobShowContextProvider } from '@/modules/client/providers/ClientSsnDobVisibility';
+import { clientSsnColumn } from '@/modules/clientMerge/components/client/NewClientMerge';
```

Comment:
Our usual pattern when exporting a column definition is to caps lock and underscore the name. I don't have a strong preference either way but it's nice to be consistent. Do you want to move towards using camel case instead?

Reply:
Agreed I will update for consistency. Originally I wasn't exporting it but I realized we have it in storybook, which was probably good to keep.